ingredient/ingredient-postgresql/README.md

```diff
@@ -0,0 +1,94 @@
+## Changelogs
+
+* [@azbake/ingredient-postgresql]
+
+## Overview
+
+The PostgreSQL ingredient is a plugin for bake. When included in a recipe, this will create an [Azure Database for PostgreSQL server](https://docs.microsoft.com/en-us/azure/postgresql/).
+This only creates the core account. Individual applications are responsible for creating and maintaining Containers / Databases / Collections.
+
+Currently this is only able to create a Flexible server. Azure is pushing users away from Single server, but if this is needed, the plugin can be enhanced to use the `src/SingleServerArm.json` template.
```

Comment:
so SingleServerArm.json is basically dead code right now? Maybe we should just remove it

Reply:
that's fine with me. I'll just add a note in the readme about how to get it again in vase we go down this road later